lib/modules/onc_program/bulk_data_group_validation_sequence.rb

```diff
@@ -382,6 +382,15 @@ def streamed_ndjson_get(url, headers)
           block_response: response_block
         )
 
+        if ['301', '302', '303', '307'].include?(response.code) && response[:location].present?
+          response = RestClient::Request.execute(
+            method: :get,
+            url: response[:location],
```

Comment:
HTTP 1.1 allows relative urls in the location header.  So you should check to see if it starts with http, and if not, calculate the relative url based on the current location.